server/test/factories.ts

```diff
@@ -336,7 +351,7 @@ export async function buildDocument(
     overrides.userId = user.id;
   }
 
-  if (!overrides.collectionId) {
+  if (!overrides.collectionId && !isNull(overrides.collectionId)) {
```

Comment:
```suggestion
  if (overrides.collectionId === undefined) {
```

Might be fine for these?

Reply:
Yes, should be good. Though ignoring other potential _falsy_ values, but it's okay.